CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This document outlines major changes between releases.
 
 ## [Unreleased]
 
+### Fixed
+- Unrestricted access to not owned objects via cache (#713)
```

Comment:
Maybe also add `Update from` section and mention new config parameters?

Reply:
I'm not sure we need such section because we don't change config params but add a new one with some default values

Reply:
You described breaking config changes. But we want to notify readers that *some* config changes are present in new release. It is important for config manager tools that should adopt configuration of new parameter somehow.  The easiest way for us to notify config manager developers is to mention it in CHANGELOG file and use `config change` label on GitHub.

We can mention it in `Fixed` section near description or add separate record in `Add` section or describe it in `Update from`. Any way is okay.